main.py

```diff
@@ -0,0 +1,16 @@
+import sys
+from parser import Parser
+from router import Router
+
+def main():
+    parser = Parser()
```

Comment:
Assuming that the program starts execution when they type in `git undo` and then the program stops afterwards, wouldn't that mean the parser is only ever used once? Could we simply make a util function instead of creating an object?